pkgs/development/tools/yarn2nix-moretea/yarn2nix/default.nix

```diff
@@ -170,6 +171,8 @@ in rec {
 
     packageGlobs = package.workspaces;
 
+    packageResolutions = package.resolutions;
```

Comment:
Does this not cause trouble when there is no `resolutions` field?

Reply:
Hm good point. It would serialize to `null` in the resultant JSON, which yarn/npm may or may not handle correctly. I switched it to default to an empty attribute set.

Reply:
Not sure how serialization matters here, what gets serialized to `null`? If the attribute doesn't exist, Nix would've thrown an error.

Reply:
Sorry, yes, you're right. But it won't now.